[조욱희] week3(week2와 통합제출)

AshtonJo opened the pull request:
### 2주차를 개인집안사정으로 완료했으나 PR하지못하여 3주차와 통합 제출합니다.
## 요구사항
- [기본] 브라우저에 현재 보이는 화면의 영역(viewport) 너비를 기준으로 분기되는 반응형 디자인을 적용했나요? [o]

PC: 1200px 이상
Tablet: 768px 이상 ~ 1199px 이하
Mobile: 375px 이상 ~ 767px 이하
375px 미만 사이즈의 디자인은 고려하지 않습니다

- [x] [랜딩 페이지] 페이스북, 카카오톡, 디스코드, 트위터 등 SNS에서 Linkbrary 랜딩 페이지(“/”) 공유 시 미리보기를 볼 수 있도록 랜딩 페이지 메타 태그를 설정했나요?

- [x] [랜딩 페이지] 미리보기에서 제목은 “Linkbrary”, 설명은 “세상의 모든 정보를 쉽게 저장하고 관리해 보세요”로 설정했나요?

- [x] [랜딩 페이지] 주소와 이미지는 자유롭게 설정했나요?

- [o] [랜딩 페이지] Tablet 사이즈에서 화면의 너비가 1199px 이하로 작아질 때 “Linkbrary” 로고와 “로그인” 버튼 사이의 간격은 변하지 않게 고정값으로 유지하되 좌우 여백이 줄어드나요?

- [o] [랜딩 페이지] Tablet 사이즈에서 최소 좌우 여백은 “Linkbrary” 로고의 왼쪽에 여백 32px, “로그인” 버튼 오른쪽 여백 32px을 유지하고“Linkbrary” 로고와 “로그인" 버튼의 간격이 가까워지나요?

- [o] [랜딩 페이지] PC, Tablet 사이즈의 이미지 크기는 고정값을 사용했나요?

- [o] [랜딩 페이지] Mobile 사이즈의 이미지는 좌우 여백 32px을 제외하고 이미지 영역이 꽉 차나요?
(이때 가로가 커지는 비율에 맞춰 세로도 커져야 합니다.)

- [x] [랜딩 페이지] Mobile 사이즈 너비가 커지면, “Privacy Policy”, “FAQ”, “codeit-2023”이 있는 영역과 SNS 아이콘들이 있는 영역의 간격이 커지나요?

- [x] [로그인, 회원가입 페이지] Tablet 사이즈에서 디자인은 PC사이즈와 동일한가요?

- [o] [로그인, 회원가입 페이지] Mobile 사이즈에서 좌우 여백 32px 제외하고 내부 요소들이 너비를 모두 차지하나요? [x]

- [o] [로그인, 회원가입 페이지] Mobile 사이즈에서 내부 요소들의 너비는 기기의 너비가 커지는 만큼 커지지만 400px을 넘지 않나요? 



### 기본

- [o] 피그마 디자인에 맞게 랜딩, 로그인, 회원가입 페이지 반응형 디자인을 만들어 주세요.
- [o] Github에 위클리 미션 PR을 만들어 주세요.
- [o] React와 같은 UI 라이브러리를 사용하지 않고 진행합니다.

### 심화

- [x] 팀원의 위클리 미션 PR에 코드 리뷰를 남겨 주세요. => 과제 제출후 올린 팀원의 코드를 본후 리뷰하겠습니다.
- [x] 랜딩 페이지, 제품 소개 영역의 mobile 사이즈에서만 제목, 이미지, 설명 순서대로 배치해 주세요.

## 주요 변경사항

- pc,mobile,tablet 버전 반응형 구현완료
- signin과 signup페이지 또한 비율에 맞게 끔 조정 완료하였습니다.

## 스크린샷


<img width="1440" alt="스크린샷 2023-11-04 오후 11 01 00" src="https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/99397079/a1525fbe-ef61-46bc-a72e-62cde4d84d92">
<img width="1440" alt="스크린샷 2023-11-04 오후 11 01 05" src="https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/99397079/df192501-653c-40d4-bbf1-687b207cc246">
<img width="1440" alt="스크린샷 2023-11-04 오후 11 01 09" src="https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/99397079/615a16ff-8a3f-4263-8d49-cae649063c1a">
<img width="1440" alt="스크린샷 2023-11-04 오후 11 01 12" src="https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/99397079/25670f94-b49a-4a5a-a115-a95894bac98d">
<img width="1440" alt="스크린샷 2023-11-04 오후 11 01 15" src="https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/99397079/c545027e-24b0-489c-8232-7406c8799855">
<img width="1440" alt="스크린샷 2023-11-04 오후 11 01 30" src="https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/99397079/a25f0510-8fa6-4c78-ba16-58519d481a06">
<img width="1440" alt="스크린샷 2023-11-04 오후 11 01 35" src="https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/99397079/6c831a65-1409-44b4-8c90-4b2390a49ff1">


## 멘토에게

- 원하는 만큼 비율 조정을 너무 못한것같습니다.. 거의 미완성급입니다 ㅠ
- section에서 column-gap을 조정했으나 비율이 안바뀌는 이유가 궁금합니다.
- 셀프 코드 리뷰를 통해 질문 이어가겠습니다.

AshtonJo committed:
2주차 내용저장

AshtonJo committed:
메인페이지 nav img 문구 수정

AshtonJo committed:
3주차 반응형 화면

AshtonJo commented:
사정상 내일 시간이 안될것같아 미완성임에도 부득이하게 제출합니당 ㅜ.ㅜ

sstaar91 commented on index.html:
```diff
@@ -0,0 +1,170 @@
+  <body>
+    <header>
+      <nav>
+        <a href="index.html">
```
```suggestion
        <a href="/">
```
루트 경로로 이동할 때는 이와같이 작성해주셔도 됩니다!

sstaar91 commented on signin.html:
```diff
@@ -0,0 +1,56 @@
+    <title>signin</title>
+  </head>
+  <body>
+    <form>
```
form 태그는 input과 button 태그를 감싸는 용도로 사용하는 것이 더 적합합니다.
현재는 모든 요소를 묶고 있는데, input과 button을 묶은 login_form에 해당 태그를 사용해주세요

sstaar91 commented on signin.html:
```diff
@@ -0,0 +1,56 @@
+          </div>
+        </div>
+      </div>
+      <a class="login" href="signin.html"> <span>로그인</span></a>
```
로그인 버튼은 button태그로 만들어 주시는게 좋을 것 같습니다.
a 태그는 다른 페이지로 이동할 때 주로 사용하는데,
해당 페이지에서는 로그인 버튼을 누르면 id와 비밀번호를 판단해 어떠한 동작을 하는 것이기 때문에
페이지 이동을 하는 a 태그와는 어울리지 않습니다.

sstaar91 commented on signin.html:
```diff
@@ -0,0 +1,56 @@
+        /></a>
+        <div class="checking_user">
+          <p class="signup">회원이 아니신가요?</p>
+          <span class="signup_right_text"
```
조금 더 명확한 class 명이 필요할 것 같습니다.
현재는 '회원가입 오른쪽 글자' 라는 의미인데, 해당 요소는 회원가입을 하러 가기 위한 링크이기 때문에
링크라는 것을 더 명확하게 드러내야 하겠습니다.

sstaar91 commented on src/signin.css:
```diff
@@ -0,0 +1,195 @@
+}
+
+input {
+  display: flex;
+  width: 400px;
+  padding: 18px 15px;
+  justify-content: center;
+  align-items: center;
+  border-radius: 8px;
+  border: 1px solid #6d6afe;
+  background-color: #ffffff;
+  margin: 12px 0;
```
- 하나의 요소에 여러가지 속성을 부여하는 경우 중요도, 
   관련도에 따라서 나름의 convention을 지켜서 작성하는 것이 좋습니다.
- 일반적인 convention 은 아래와 같습니다. 아래와 같이 순서 적용해주세요.
    
    [CSS property 순서]
    
    - Layout Properties (position, float, clear, display)
    - Box Model Properties (width, height, margin, padding)
    - Visual Properties (color, background, border, box-shadow)
    - Typography Properties (font-size, font-family, text-align, text-transform)
    - Misc Properties (cursor, overflow, z-index)

sstaar91 commented on src/signup.css:
```diff
@@ -0,0 +1,122 @@
+body {
+  display: flex;
+  flex-direction: column;
+  width: 1440px;
```
해상도가 1440보다 큰 해상도에서는 배경색이 잘릴 수도 있습니다.
body 태그에 고정된 값을 적용할 때는 브라우저의 차이도 고려해서 적용해주세요

sstaar91 commented on src/style.css:
```diff
@@ -0,0 +1,317 @@
+    height: 300px;
+  }
+
+  .slogan {
+    font-size: 60px;
+  }
+  .section {
```
```suggestion
  .slogan {
    font-size: 60px;
  }
  
  .section {
```
선택자끼리는 가독성을 위해서 한칸씩 띄워주세요!

sstaar91 commented on src/style.css:
```diff
@@ -0,0 +1,317 @@
+  .title {
+    grid-area: "title";
+    font-size: 37px;
+    text-align: center; /* 가운데 정렬 */
+    margin-bottom: -50px; /* 아래 마진 추가 */
+    line-height: 50px;
+  }
+
+  .description {
+    grid-area: "description";
+    font-size: 16px;
+    text-align: center; /* 가운데 정렬 */
```
스타일 옆에 적어두신 주석은 제거해주세요

주석을 남기는 것은 복잡한 코드에서 설명이 필요하다고 생각되는 부분이나
차후 수정 혹은 추가를 위해 남겨두는 상황을 제외하고는
주석을 사용하는 것을 지양해주세요

sstaar91 reviewed (commented):
욱희님 수고하셨습니다!👍
아직 미완성인 부분이 있는 것 같네요!
그래서 리뷰의 양이 적다고 생각할 수 있는데, 현재 남겨놓은 리뷰는 해당 파일의
해당 줄에만 국한되는 것이 아닌, 다른 파일의 코드에서도 적용이 되어야 하는 부분도 있습니다.

남겨놓은 리뷰를 보고, 자신의 코드를 보면서 리뷰가 남겨지지 않은 부분이라고 할지라도
해당 리뷰가 동일하게 적용되어야 한다면 반드시 수정해주세요!

AshtonJo committed:
수정사항
1. 메타 태그 추가
2. form 태그 위치 변경: header 포함 전체를 감싸던 태그를 input과 button을 감싸게 수정
3. 로그인버튼 역할을 하는 a 태그 => button 태그로 수정
4. signup_right_text 클래스명을 signin.html=> signupLink 로 수정, signup.html=> loginLink 로 클래스명 각각 수정
5. siginin.css, signup.css => CSS PROPERTY 순서 변경
6. 모든파일의 body 태그 width를 1440px => width: 100%로 수정
7. 선택자끼리의 가독성을 위해 한칸씩 띄워 수정
8. 불필요 주석 제거

AshtonJo committed:
footer 수정
